src/common/crypto/platform/openssl/crypto.cpp

```diff
@@ -282,8 +283,15 @@ ExpectedPrivateKey LoadFrom(const Args &args) {
 #endif // ndef MENDER_CRYPTO_OPENSSL_LEGACY
 
 ExpectedPrivateKey PrivateKey::Load(const Args &args) {
+	// Numerous internal OpenSSL functions call OPENSSL_init_ssl().
+	// Therefore, in order to perform nondefault initialisation,
+	// OPENSSL_init_ssl() MUST be called by application code prior to any other OpenSSL function
+	// calls. See: https://docs.openssl.org/1.1.1/man3/OPENSSL_init_ssl/#description
```

Comment:
Could we link to a more [recent version](https://docs.openssl.org/3.0/man3/OPENSSL_init_ssl/)?